src/pages/login/index.tsx

```diff
@@ -1,5 +1,28 @@
+import { ArrowLeft } from '../../assets';
+import { AppBar } from '../../ui/app-bar';
+import { ButtonText } from '../../ui/button-text';
 import { PageLayout } from '../../ui/layout/page-layout';
+import { LoginSection } from './loginSection';
+import * as styles from './styles.css';
 
 export const LoginPage = () => {
-  return <PageLayout>LoginPage</PageLayout>;
+  return (
+    <PageLayout>
+      <AppBar left={<ArrowLeft />} variant="page"></AppBar>
```

Comment:
arrow back 버튼에 useNavigate로 이전 페이지로 가게 하거나 onboading 페이지로 이동하도록 하는 이벤트 핸들러를  추가해야 할 것 같습니당. 그리고 헤더에 이렇게 넣도록 설계 했는데 태구님이 하신대로 넣어도 동작은 할 것 같네요.

```suggestion
    <PageLayout header={<AppBar left={<ArrowLeft />} variant="page"/>}>
```